[BUG] Creating Next.js App as described in Docs FAILS

### Describe the bug

Hello, I wanted to test this and tried running the command from the get started section in the [docs](https://opennext.js.org/cloudflare/get-started)
`npm create cloudflare@latest -- my-next-app --framework=next --experimental` 
and this happens: 
```npm error peer wrangler@"^3.111.0" from @opennextjs/cloudflare@0.5.11
npm error node_modules/@opennextjs/cloudflare
npm error   dev @opennextjs/cloudflare@"0.5.x" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /Users/big/.npm/_logs/2025-03-15T15_15_46_440Z-eresolve-report.txt
npm error A complete log of this run can be found in: /Users/big/.npm/_logs/2025-03-15T15_15_46_440Z-debug-0.log

│
╰  ERROR  Error: npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: my-next-app-3@0.1.0
npm error Found: wrangler@4.0.0
npm error node_modules/wrangler
npm error   dev wrangler@"^4.0.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peer wrangler@"^3.111.0" from @opennextjs/cloudflare@0.5.11
npm error node_modules/@opennextjs/cloudflare
npm error   dev @opennextjs/cloudflare@"0.5.x" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error```

Using node v22.14.0 on macOS

### Steps to reproduce

Run ```npm create cloudflare@latest -- my-next-app --framework=next --experimental```

### Expected behavior

I expected it not to fail. 

### @opennextjs/cloudflare version

0.5 I guess

### Wrangler version

4

### next info output

```shell
Not possible
```

### Additional context

_No response_

--- a/.changeset/orange-tools-approve.md
+++ b/.changeset/orange-tools-approve.md
@@ -1,0 +1,5 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+Update wrangler dependency

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -33,7 +33,7 @@ catalog:
   typescript-eslint: ^8.7.0
   typescript: ^5.7.3
   vitest: ^2.1.1
-  wrangler: ^3.111.0
+  wrangler: "^3.114.1 || ^4.0.0"
 
 # e2e tests
 catalogs:

